DOCU-2414: Adds Get Kong to getting started guide (#4284)
Kong/docs.konghq.com: src/gateway/get-started/get-kong.md

   ```text
   ✔ Kong is ready!
   ```

1. Verify that {{site.base_gateway}} is running:

   {{site.base_gateway}} serves an Admin API on the default port `8001`. The Admin API can be used for
   both querying and controlling the state of {{site.base_gateway}}. The following command
   will query the Admin API, fetching the headers only:

   ```sh
   curl --head localhost:8001
   ```

   If {{site.base_gateway}} is running properly, it will respond with a `200` HTTP code, similar to the following: 

   ```text
   HTTP/1.1 200 OK
   Date: Mon, 22 Aug 2022 19:25:49 GMT
   Content-Type: application/json; charset=utf-8
   Connection: keep-alive
   Access-Control-Allow-Origin: *
   Content-Length: 11063
   X-Kong-Admin-Latency: 6
   Server: kong/{{page.kong_version}}
   ```

1. Evaluate the {{site.base_gateway}} configuration:

   The root route of the Admin API provides important information about the running 
   {{site.base_gateway}} including networking, security, and plugin information. The full 
   configuration is provided in the `.configuration` key of the returned JSON document.

   ```sh
   curl -s localhost:8001 | jq '.configuration'
   ```

>    You should receive a large JSON document with various {{site.base_gateway}} configuration information.

Reviewer: ```suggestion
   You should receive a large JSON response with {{site.base_gateway}} configuration information.
```

---
title: Get Kong
content-type: tutorial
---

[{{site.base_gateway}}](/gateway/latest/) is a lightweight, fast, and flexible cloud-native API gateway. 
{{site.base_gateway}} sits in front of your service applications, dynamically controlling, analyzing, and 
routing requests and responses. {{site.base_gateway}} implements your API traffic policies 
by using a flexible, low-code, plug-in based approach. 

This tutorial will help you get started with {{site.base_gateway}} by setting up a local installation
and walking through some common API management tasks. 

This page will walk you through running {{site.base_gateway}} and verifying it with the
[Admin API](/gateway/latest/admin-api). Once complete, the following tasks 
can be performed to complete the tutorial:

> * [Understanding and configuring Services and Routes](/gateway/latest/get-started/configure-services-and-routes)
> * [Configuring Rate Limiting to protect upstream Services](/gateway/latest/get-started/configure-ratelimiting)
> * [Increase system performance with Proxy Caching](/gateway/latest/get-started/proxy-caching)
> * [Load Balancing for horizontal Service scaling](/gateway/latest/get-started/load-balancing)
> * [Protecting Services with Key Authentication](/gateway/latest/get-started/protect-services)
> * [Use Workspaces and Teams to further protect {{site.base_gateway}}](/gateway/latest/get-started/manage-teams)

Reviewer: ```suggestion
* [Understanding and configuring Services and Routes](/gateway/latest/get-started/configure-services-and-routes)
* [Configuring Rate Limiting to protect upstream Services](/gateway/latest/get-started/configure-ratelimiting)
* [Increase system performance with Proxy Caching](/gateway/latest/get-started/proxy-caching)
* [Load Balancing for horizontal Service scaling](/gateway/latest/get-started/load-balancing)
* [Protecting Services with Key Authentication](/gateway/latest/get-started/protect-services)
* [Use Workspaces and Teams to organize {{site.base_gateway}}](/gateway/latest/get-started/manage-teams) at scale.
```

Reviewer: I took this from our marketing site, I felt it was a bit stronger than 'further protect'.

   initialized with the appropriate [migration](/gateway/latest/reference/cli/#kong-migrations) steps, 
   and once the {{site.base_gateway}} is ready, you will see the following message:

   ```text
   ✔ Kong is ready!
   ```

1. Verify that {{site.base_gateway}} is running:

   {{site.base_gateway}} serves an Admin API on the default port `8001`. The Admin API can be used for
   both querying and controlling the state of {{site.base_gateway}}. The following command
   will query the Admin API, fetching the headers only:

   ```sh
   curl --head localhost:8001
   ```

   If {{site.base_gateway}} is running properly, it will respond with a `200` HTTP code, similar to the following: 

   ```text
   HTTP/1.1 200 OK
   Date: Mon, 22 Aug 2022 19:25:49 GMT
   Content-Type: application/json; charset=utf-8
   Connection: keep-alive
   Access-Control-Allow-Origin: *
   Content-Length: 11063
   X-Kong-Admin-Latency: 6
   Server: kong/{{page.kong_version}}
   ```

1. Evaluate the {{site.base_gateway}} configuration:

   The root route of the Admin API provides important information about the running 
   {{site.base_gateway}} including networking, security, and plugin information. The full 

>    configuraiton is provided in the `.configuration` key of the returned JSON document.

Reviewer: ```suggestion
   configuration is provided in the `.configuration` key of the returned JSON document.
```

   ```sh
   curl -s localhost:8001 | jq '.configuration'
   ```

   You should receive a large JSON document with various {{site.base_gateway}} configuration information.

> The following steps in this tutorial depend on a running {{site.base_gateway}}, so leave

Reviewer: ```suggestion
Every step in this tutorial requires a running {{site.base_gateway}}, so leave
```
